roles/vdm/tasks/deploy.yaml

```diff
@@ -1,86 +1,86 @@
 ---
-
-- name: Create namespace
+- name: deploy - Apply SAS Viya deployment
   community.kubernetes.k8s:
```

Comment:
Guessing this was not converted over in this PR from community -> kubernetes

Reply:
This is going to be merged and released before https://github.com/sassoftware/viya4-deployment/pull/338 IAC-770 where that ansible collection modules is initially updated so that PR will need to rebase + update before release.